### PR TITLE
Update Usage.md to explain usage of encrypted database

### PR DESCRIPTION
**Pull request by woehrl01**

### CODE REVIEW

File: Usage.md [hunk 1]
```diff
@@ -106,4 +106,33 @@ config.setSharedCache(true);
...
+*__Important: xerial/sqlite-jdbc does not support encryption out of the box, you need a special .dll/.so__*
+
+SQLite support encryption of the database via special drivers and a key. To use an encrypted database you need a driver which supports encrypted database via `pragma key` or `pragma hexkey`, e.g. SQLite SSE or SQLCipher. You need to specify those drivers via directly referencing the .dll/.so through:
+
+```
+-Dorg.sqlite.lib.path=.
+-Dorg.sqlite.lib.name=sqlite_cryption_support.dll
+```
+
+Now the only need to specify the passwort is via:
```
Comment (PromanSEW) [inline Usage.md]: passwor**d**

File: Usage.md [hunk 2]
```diff
@@ -106,4 +106,33 @@ config.setSharedCache(true);
...
+SQLite support encryption of the database via special drivers and a key. To use an encrypted database you need a driver which supports encrypted database via `pragma key` or `pragma hexkey`, e.g. SQLite SSE or SQLCipher. You need to specify those drivers via directly referencing the .dll/.so through:
+
+```
+-Dorg.sqlite.lib.path=.
+-Dorg.sqlite.lib.name=sqlite_cryption_support.dll
+```
+
+Now the only need to specify the password is via:
+```java
+Connection connection = DriverManager.getConnection("jdbc:sqlite:db.sqlite", "", "S3cr3T");
```
Comment (xerial) [inline Usage.md]: Is this a password?

Comment (xerial) [reply Usage.md]: If so `(password)` might be better

**Comment by xerial** [comment]
Merged. Thanks for the update!

**Pull request merged**